### PR TITLE
Batch consume msg in rockmq internal loop

### LINKED ISSUE #11372: [Enhancement]: Batch consume for rmq internal loop
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What would you like to be added?

Allow rocksmq internal consume loop perform batch Consumer on rocksdb.

### Why is this needed?

When doing fast forwarding on a topic, each consume call with size of 1 will lead to lots of random read in rockdb, which will damage the efficiency of the system

### Anything else?

Related to #7313

### PR DESCRIPTION
**Pull request by congqixia**
Resolves #11372
See also #7313 

/kind enhancement

/cc @fishpenguin 

Signed-off-by: Congqi Xia <congqi.xia@zilliz.com>

### CODE REVIEW

File: internal/util/rocksmq/server/rocksmq/rocksmq_impl.go [hunk 1]
```diff
@@ -776,20 +786,22 @@ func (rmq *rocksmq) updateAckedInfo(topicName, groupName string, newID UniqueID,
...
+	err = rmq.kv.Save(beginIDKey, strconv.FormatInt(lastID, 10))
 	if err != nil {
 		return err
 	}
 
 	// Update begin_id for topic
 	if vals, ok := rmq.consumers.Load(topicName); ok {
-		var minBeginID int64 = -1
+		var minBeginID int64 = math.MaxInt64
 		for _, v := range vals.([]*Consumer) {
```
Comment (czs007) [inline internal/util/rocksmq/server/rocksmq/rocksmq_impl.go:805]: Could vals be empty?

Comment (fishpenguin) [reply internal/util/rocksmq/server/rocksmq/rocksmq_impl.go:805]: Couldn't.

Comment (congqixia) [reply internal/util/rocksmq/server/rocksmq/rocksmq_impl.go:805]: @czs007 there must be a consumer associated with this `topicName` to call this function.

**Pull request merged**